docker/package/package_generator.py

```diff
@@ -39,6 +40,12 @@
     help="specify directory with source archive(s) for ubuntu packages",
     type=os.path.abspath,
 )
+parser.add_argument(
+    "--launchpad-sources",
+    help="download sources from launchpad if not provided",
```

Comment:
If these are mutually exclusive this description is not correct:
```suggestion
    help="download sources from launchpad instead of providing them",
```